builder/packages.sls

```diff
@@ -26,3 +26,34 @@ builder-packages:
           - libllvm4.0
           - libllvm4.0:i386
           - llvm-4.0-dev
+          - curl
+            # Android lists this as a dependency
+          - distcc
+          - dosfstools
+          - gettext
+          - git-core
```

Comment:
I think this will be pulled in with git from slave

Reply:
I don't think `git` pulls in `git-core`, at least the debian package page for git doesn't list git-core as a dependency: https://packages.debian.org/sid/git

Reply:
Okay, ignore this comment then :)